ui/pages/unlock-page/unlock-page.component.js

```diff
@@ -246,96 +239,78 @@ export default class UnlockPage extends Component {
               {this.renderMascot()}
               {isBeta() ? (
-                <Box className="unlock-page__mascot-container__beta">
+                <Text
+                  className="unlock-page__mascot-container__beta"
+                  backgroundColor={BackgroundColor.primaryDefault}
+                  color={TextColor.primaryInverse}
+                  padding={1}
+                  borderRadius={BorderRadius.LG}
+                  textTransform={TextTransform.Uppercase}
+                  fontWeight={FontWeight.Medium}
+                >
                   {t('beta')}
-                </Box>
+                </Text>
               ) : null}
             </Box>
             <Text
               data-testid="unlock-page-title"
               as="h1"
-              variant={TextVariant.headingLg}
-              marginTop={1}
-              marginBottom={2}
+              variant={TextVariant.displayMd}
+              marginBottom={6}
+              fontWeight={FontWeight.Medium}
               color={TextColor.textDefault}
+              textAlign={TextAlign.Center}
             >
               {t('welcomeBack')}
             </Text>
             <FormTextField
               id="password"
-              label={
-                <Box
-                  display={Display.Flex}
-                  width={BlockSize.Full}
-                  justifyContent={JustifyContent.spaceBetween}
-                  alignItems={AlignItems.center}
-                  marginBottom={1}
-                >
-                  <Text variant={TextVariant.bodyMdMedium}>
-                    {t('password')}
-                  </Text>
-                </Box>
-              }
-              placeholder={t('enterPassword')}
+              placeholder={t('enterYourPassword')}
               size={FormTextFieldSize.Lg}
               inputProps={{
                 'data-testid': 'unlock-password',
-                type: InputType.Password,
+                'aria-label': t('password'),
               }}
               onChange={(event) => this.handleInputChange(event)}
+              type={TextFieldType.Password}
+              value={password}
               error={Boolean(error)}
               helpText={this.renderHelpText()}
               autoComplete
               autoFocus
               disabled={isLocked}
               width={BlockSize.Full}
-              textFieldProps={{
-                borderRadius: BorderRadius.LG,
-              }}
+              marginTop={4}
+              marginBottom={4}
             />
-          </Box>
-          <Box
-            display={Display.Flex}
-            flexDirection={FlexDirection.Column}
-            width={BlockSize.Full}
-            alignItems={AlignItems.center}
-          >
-            <Box
-              className="unlock-page__buttons"
-              display={Display.Flex}
-              flexDirection={FlexDirection.Column}
-              width={BlockSize.Full}
-              gap={4}
```

Comment:
Removing some unneeded html nesting and bloat here the button html should remain the same